awxkit/awxkit/api/pages/api.py

```diff
@@ -268,9 +268,9 @@ def _assign_roles(self):
     def _assign_related(self):
         for _page, name, related_set in self._related:
             endpoint = _page.related[name]
-            if isinstance(related_set, dict):  # Relateds that are just json blobs, e.g. survey_spec
+            if isinstance(related_set, dict):  # Related that are just json blobs, e.g. survey_spec
```

Comment:
The original here wasn't a spelling mistake.  Maybe "Related objects that are..." would be clearer, though?

Reply:
Yes, possibly.